e2e/wdio/core/tests/object-number.e2e-spec.ts

```diff
@@ -54,23 +54,23 @@ describe('object number test suite', function () {
     });
 
     describe('basic object number examples', function () {
-        it('should check colors and values', () => {
+        xit('should check colors and values', () => {
```

Comment:
wouldn't it be easier if we disable the whole block starting on line 25?

Reply:
there are not only color checks